planter/planter.sh

```diff
@@ -24,7 +24,7 @@ IMAGE="${IMAGE_NAME}:${TAG}"
 # run our docker image as the host user with bazel cache and current repo dir
 REPO=$(git rev-parse --show-toplevel 2>/dev/null || true)
 REPO=${REPO:-${PWD}}
-VOLUMES="-v ${REPO}:${REPO} -v ${HOME}:${HOME} --tmpfs /tmp:exec,mode=777"
+VOLUMES="-v ${REPO}:${REPO} -v ${HOME}:${HOME} --mount type=tmpfs,destination=${HOME}/.docker --tmpfs /tmp:exec,mode=777"
```

Comment:
this change probably wants a comment explaining it.

also what's the difference between `--mount type=tmpfs` and `--tmpfs`?

Reply:
SGTM

`--mount` is what docker's docs would prefer you to use, but `--tmpfs` actually allows you to set `:exec` 🤷‍♂️

Reply:
I rewrote all of the comments, they were a bit stale and it seemed worth more generally explaining why we do things for anyone curious.